docs/LV_client/tutorials/Logger.md

```diff
@@ -38,11 +43,17 @@ This VI will produce a text file for:
 <img class="lg_img" src="../logger.png"/>
 </p>
 
-## Mono_parse_binaries.vi
+## mono_parse_binaries.vi
+
+To visualize data from a binary file:
+
+1. Browse to the configuration file (this will be the `.txt` file created by the logger.vi) to be parsed.
+2. The program will find the correspoding binary file.   
+
+### Camera data
+For camera images the user needs to specify a path to save the image. If the folder does not exist, the folder will be created.
 
-This tool can help the user to visualize data from a binary file.
+Then the user will need to select the file format if he/she wants to save as a JPEG or PNG formatted image. By default the image file format is saved as PNG.
```

Comment:
One more suggestion here, just needs an "it". Then the user will need to select the file format if he/she wants to save it as a JPEG or PNG formatted image.